### PR TITLE
Testing: Check python3 compatibility; Fix #1505

### LINKED ISSUE #1505: Python3 compatibility test
Motivation
----------
Many files are now python3 compatible. It would be good to have a flag to identify them. The files found as python3 compatible must stay compatible. Any PR submitted that would break the compatibility should be automatically reject by Travis

### PR DESCRIPTION
**Pull request by hahahannes**
Testing: Check python3 compatibility; Fix #1505

### CODE REVIEW

File: tools/travis/check_python_3.sh [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+# limitations under the License.
+#
+# Authors:
+# - Hannes Hansen <hannes.jakob.hansen@cern.ch>, 2018
+
+echo '==============================='
+echo 'Running pylint                 '
+echo '==============================='
+
+for filename in $(find lib/rucio bin);
```
Comment (bari12) [inline tools/travis/check_python_3.sh]: Maybe only test files named in `changed_files.txt` similar to the check_syntax test.

**Comment by bari12** [comment]
You also have to adapt the install script: travis/install_script.sh to at least install the pip-requires-test, otherwise you won't have pylint in the environment.

File: tools/travis/py3_compatible.txt [hunk 2]
```diff
@@ -0,0 +1,107 @@
+bin
```
Comment (bari12) [inline tools/travis/py3_compatible.txt]: This one should not be part of the commit I think 😃

File: tools/travis/check_python_3.sh [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+# - Hannes Hansen <hannes.jakob.hansen@cern.ch>, 2018
+
+echo '==============================='
+echo 'Running pylint                 '
+echo '==============================='
+
+for filename in $(cat changed_files.txt);
+do
+    if grep -q "PY3K COMPATIBLE" $filename; then
+        echo "Check if file" $filename "is compatible"
```
Comment (mlassnig) [inline tools/travis/check_python_3.sh]: change string to " ... is Python3 compatible .." ?

**Review by bari12 — APPROVED** [review]
